doc/developer-notes.md

```diff
@@ -282,6 +283,20 @@ configure option adds `-DDEBUG_LOCKORDER` to the compiler flags. This inserts
 run-time checks to keep track of which locks are held and adds warnings to the
 `debug.log` file if inconsistencies are detected.
 
+### DEBUG_ADDRMAN
+
+Defining `DEBUG_ADDRMAN` performs frequent (and expensive) consistency checks on
+the entire addrman data structure. Add this line to enable it, e.g. in
```

Comment:
is `src/addrman.h` the only file we could define this in? Are there other files we could do this in. If there are I feel as though we could improve the note here. Otherwise, we could remove `e.g`.

Reply:
Good point. Improved as follows:

```diff
 Defining `DEBUG_ADDRMAN` performs frequent (and expensive) consistency checks on
-the entire addrman data structure. Add this line to enable it, e.g. in
-`src/addrman.h`
+the entire addrman data structure. To enable it, add this line in `src/addrman.h`
+before `void Check()`
```